gateway/api/v1/serializers.py

```diff
@@ -60,10 +63,8 @@ def validate(self, attrs):
                 provider = title_split[0]
             provider_instance = Provider.objects.filter(name=provider).first()
             if provider_instance is None:
-                raise ValidationError(
-                    "Custom images are only available a valid provider."
-                )
-            if not provider_instance.registry or not image.startswith(
+                raise ValidationError(f"{provider} is not valid provider.")
+            if provider_instance.registry and not image.startswith(
```

Comment:
Are we sure here that `provider_instance` always has `registry` attribute?

Reply:
you mean if the `registry` from `provider_instance` can be `None`? The answer is yes. What I'm trying to do here is that if it's `None` it's ok, we accept any value in the image.

Reply:
Until now we were forcing it to have a value but there are situations where it can has no sense like in local development.